app/views/feedback_forms/new.html.erb

```diff
@@ -1 +1,6 @@
-<%= render_feedback_form(@form_type) %>
+
+<% if @form_type == "connection" %>
+    <%= render_feedback_form("connection") %>
+<% else %>
```

Comment:
Can we add an `elsif request.xhr?` (assuming you've rebased) here that just draws the form (no modal stuff) here?  I feel this can simplify a lot of the css and remove a bunch of conditional logic.

Reply:
I am working on that - this was just me squashing commits :)